packages/grafana-ui/src/components/Layout/Box/Box.tsx

```diff
@@ -136,7 +136,7 @@ const customBorderColor = (color: BorderColor, theme: GrafanaTheme2) => {
     case 'success':
     case 'info':
     case 'warning':
-      return theme.colors[color].border;
+      return theme.colors[color].borderTransparent;
```

Comment:
We might need to use `.border` at some point. We could add more types to the `borderColor` options that are `infoTransparent`, `warningTransparent`, ... and then have more cases here in the switch to point at those colors.

Reply:
@JoaoSilvaGrafana Since the background was always using the transparent background I think it's fine for now, but definitely a problem for the future if we want to support both. hm... tricky we could add a "solid" option prop in the future that switches the background to use .main and border to user border. But this requires changing the styling functions a bit so we can pass in another prop as a dependency for how to process the border & background